@xen-orchestra/web/src/components/site/pools/PoolSidePanel.vue

```diff
@@ -0,0 +1,243 @@
+        accent="brand"
+        :left-icon="faEdit"
+      >
+        {{ t('change-state') }}
+      </UiButton>
+      <UiButton
+        v-tooltip="t('coming-soon')"
+        disabled
+        variant="tertiary"
+        size="medium"
+        accent="danger"
+        :left-icon="faTrash"
+      >
+        {{ t('forget') }}
+      </UiButton>
+      <UiButtonIcon v-tooltip="t('coming-soon')" disabled accent="brand" size="medium" :icon="faEllipsis" />
+    </template>
+    <template #default>
+      <UiCard v-if="server.error === undefined" class="card-container">
+        <UiCardTitle>
+          {{ t('general-information') }}
+        </UiCardTitle>
+        <div class="content">
+          <!-- Pool -->
+          <VtsCardRowKeyValue>
+            <template #key>{{ t('pool') }}</template>
+            <template #value>
+              <UiLink
+                v-if="server.poolId !== undefined && server.poolNameLabel !== undefined"
+                :icon="faCity"
+                size="small"
+                :to="`/pool/${server.poolId}/`"
+              >
+                {{ server.poolNameLabel }}
+              </UiLink>
+            </template>
+            <template v-if="server.poolId !== undefined" #addons>
+              <VtsCopyButton :value="server.poolId" />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- ID -->
+          <VtsCardRowKeyValue>
+            <template #key>{{ t('id') }}</template>
+            <template #value>{{ server.id }}</template>
+            <template #addons>
+              <VtsCopyButton :value="server.id" />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- Description -->
+          <VtsCardRowKeyValue>
+            <template #key>{{ t('description') }}</template>
+            <template #value>{{ server.poolNameDescription }}</template>
+            <template v-if="server.poolNameDescription !== undefined" #addons>
+              <VtsCopyButton :value="server.poolNameDescription" />
+            </template>
+          </VtsCardRowKeyValue>
+          <!-- tag -->
+          <VtsCardRowKeyValue>
+            <template #key>{{ t('tags') }}</template>
+            <template #value>
+              <UiTagsList v-if="pool !== undefined && pool.tags.length > 0">
+                <UiTag v-for="tag in pool.tags" :key="tag" accent="info" variant="primary">{{ tag }}</UiTag>
+              </UiTagsList>
+            </template>
+            <template v-if="pool !== undefined && pool.tags.length > 0" #addons>
+              <VtsCopyButton :value="pool.tags.join(', ')" />
+            </template>
+          </VtsCardRowKeyValue>
+        </div>
+      </UiCard>
+      <UiAlert v-else accent="danger">
+        {{ t('connection-failed') }}
+        <template #description>
+          {{ t('unable-to-connect-to-the-pool') }}
+        </template>
+      </UiAlert>
+      <UiCard class="card-container">
+        <UiCardTitle class="typo-body-bold text-ellipsis">
```

Comment:
The typo class is not necessary. Also, if we want to ellipsis the title, we should have a tooltip on this component:
```suggestion
        <UiCardTitle class="text-ellipsis">
```